src/Microsoft.Identity.Client/ApiConfig/SystemWebViewOptions.cs

```diff
@@ -68,5 +73,85 @@ internal void LogParameters(ICoreLogger logger)
             logger.InfoPii("BrowserRedirectError " + BrowserRedirectError,
                "BrowserRedirectError? " + (BrowserRedirectError != null));
         }
+
+#if NET_CORE || DESKTOP || NETSTANDARD
+
+        /// <summary>
+        /// Use Microsoft Edge to navigate to the given uri. On non-windows platforms it uses 
```

Comment:
What happens on older versions of windows where there is no Edge? Does it use IE?

Reply:
No idea. This is a reason why I don't really want to provide these helpers out of the box - ensuring a good experience (e.g. fallback to IE) is not easy.